src/main/java/com/aws/greengrass/device/ClientDevicesAuthService.java

```diff
@@ -66,6 +66,9 @@ public class ClientDevicesAuthService extends PluginService {
     public static final String CA_PASSPHRASE = "ca_passphrase";
     public static final String CERTIFICATES_KEY = "certificates";
     public static final String AUTHORITIES_TOPIC = "authorities";
+    public static final String SETTINGS_TOPIC = "settings";
```

Comment:
not sure about this name. All of the config is settings isn't it.

Reply:
if not "tuning", then perhaps "performance"?

Reply:
I agree. Let me circle back with a better name. I'm okay with `tuning`.

Reply:
Update?